tests/test_admin_sync.py

```diff
@@ -217,6 +247,13 @@ def test_list_page_permission_actions(client: TestClient) -> None:
     assert response.text.count('<i class="fa-solid fa-pencil"></i>') == 0
     assert response.text.count('<i class="fa-solid fa-trash"></i>') == 10
 
+    response = client.get("/admin/profile/list")
+
+    assert response.status_code == 200
+    assert response.text.count('<i class="fa-solid fa-eye"></i>') == 10
+    assert response.text.count('<i class="fa-solid fa-pencil"></i>') == 0
+    assert response.text.count('<i class="fa-solid fa-trash"></i>') == 10
```

Comment:
Do we need these tests?
I've been thinking on even reducing these tests in `test_admin_sync.py` and `test_admin_async.py` and at least I could not see the point. For example this is testing the list page which already is tested by two other models. What do you think?

Reply:
Ok, I'll remove them.